src/main/java/backend/Model.java

```diff
@@ -124,6 +129,6 @@ public NN_Model(Instances d, String[] params) throws Exception {
     protected void run() throws Exception {
```

Comment:
I have assembled these but in order to use them I'll need to try...catch {throw} them. 

Is that something we want to do?